src/main/java/net/sf/jabref/model/cleanup/FieldFormatterCleanup.java

```diff
@@ -24,8 +27,10 @@ public FieldFormatterCleanup(String field, Formatter formatter) {
 
     @Override
     public List<FieldChange> cleanup(BibEntry entry) {
-        if ("all".equalsIgnoreCase(field)) {
+        if (FieldName.ABSTRACT_ALL_FIELD.equalsIgnoreCase(field)) {
             return cleanupAllFields(entry);
```

Comment:
Is it possible to add test coverage here? - This branch is not covered by test cases.